examples/niscope/graph-measurement-betterproto.py

```diff
@@ -60,13 +60,17 @@
 #set to false to disable graph and increase speed
 show_plot = True 
 
+# Handle closing of plot window.
+closed = False
+def on_close(event):
+    global closed
+    closed = True
+
 # Error Reporting
 async def CheckStatus(scope_service, vi, result):
     if (result.status != 0):
-        error_result = await scope_service.get_error_message(vi = vi, error_code = result.status)
-        print(error_result.error_message)
-        exit()
-
+        error_result2 = await scope_service.get_error(vi = vi)
```

Comment:
@ni-pkhare , why get error is being used instead of get error message which was there earlier?

Reply:
GetError returns additional information about the error which is much more descriptive. Thats why I used it.

Reply:
Rename from `error_result2` to `error_result`

Reply:
But get error does not seem to take error code as input.  I am guessing it might return most recent error which happened. This might be dangerous in multi client case. Can you confirm that behaviour once again? And if that's the case, please change it any other error api which takes error code also as input

Reply:
@harsha-bhushan , what would you prefer? The older error api or the new api which does not take error code?

Reply:
I have changed it back to original getErrorMessage API.